Probesys/bileto
# imp: Improve the look of the email notifications

## Related issue(s)



https://github.com/Probesys/bileto/issues/447

## How to test manually



1. Ask  to reset your password
2. Create a ticket
3. Answer to a ticket with two different actors
4. Verify that for each action, you receive an email and that the HTML and TXT versions look good

## Checklist
